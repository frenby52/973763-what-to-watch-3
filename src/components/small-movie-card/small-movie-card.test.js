import React from "react";
import renderer from "react-test-renderer";
import SmallMovieCard from "../small-movie-card/small-movie-card.jsx";

const titleItem = `Fantastic Beasts`;

it(`SmallMovieCard component should render correct`, () => {
  const tree = renderer
    .create(<SmallMovieCard title={titleItem} onTitleClick={() => {}} />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
