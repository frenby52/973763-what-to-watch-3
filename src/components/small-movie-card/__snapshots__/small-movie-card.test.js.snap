// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallMovieCard component should render correct 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onClick={[Function]}
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <Link
    style={
      Object {
        "color": "unset",
      }
    }
    to="/films/0"
  >
    <div
      className="small-movie-card__image"
    >
      <video
        height="175"
        width="280"
      />
    </div>
    <h3
      className="small-movie-card__title"
    >
      <span
        className="small-movie-card__link"
      >
        The Grand Budapest Hotel
      </span>
    </h3>
  </Link>
</article>
`;
